<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindDialog</class>
 <widget class="QDialog" name="FindDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>136</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find</string>
  </property>
  <widget class="QPushButton" name="findButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>8</y>
     <width>96</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="matchwordCheck">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>96</y>
     <width>174</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Match whole word only</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>96</y>
     <width>96</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="matchcaseCheck">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>72</y>
     <width>174</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Match Case</string>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="findEdit">
   <property name="geometry">
    <rect>
     <x>56</x>
     <y>8</y>
     <width>256</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="replaceButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>96</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Replace</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="findLabel">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>8</y>
     <width>48</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Find:</string>
   </property>
  </widget>
  <widget class="QWidget" name="directionWidget" native="true">
   <property name="geometry">
    <rect>
     <x>184</x>
     <y>72</y>
     <width>128</width>
     <height>48</height>
    </rect>
   </property>
   <widget class="QRadioButton" name="directionUp">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>24</y>
      <width>64</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Up</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="directionDown">
    <property name="geometry">
     <rect>
      <x>64</x>
      <y>24</y>
      <width>64</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Down</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="directionLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>128</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Direction</string>
    </property>
    <property name="indent">
     <number>2</number>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="replaceEdit">
   <property name="geometry">
    <rect>
     <x>56</x>
     <y>40</y>
     <width>256</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="replaceLabel">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>40</y>
     <width>48</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Replace:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="replaceAllButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>68</y>
     <width>96</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Replace All</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
